tools/docs/generate.go

```diff
@@ -49,21 +49,55 @@ func markdown(path string, cmd *cobra.Command) {
 	must(doc.GenMarkdownTree(cmd, path))
 }
 
+func man(path string, header *doc.GenManHeader, cmd *cobra.Command) {
+	must(os.MkdirAll(path, 0755))
+	must(doc.GenManTree(cmd, header, path))
+}
+
+type commandWithManHeader struct {
```

Comment:
Nit: we can just call this "command", e.g.

```Go
type commandWithManHeader struct {
	cmd *cobra.Command
	man  *doc.GenManHeader
}
```

Reply:
addressed